docs/src/@primer/gatsby-theme-doctocat/nav.yml

```diff
@@ -68,7 +68,7 @@
   - title: Label
     url: "/components/label"
   - title: Layout
-    url: "/components/layout"
+    url: "/components/alpha/layout"
```

Comment:
this didn't get updated when layout was moved into the alpha folder, and that was causing old docs to show up on the site (production is currently showing old docs)

Reply:
🤔 I wonder if we could find a way to automatically check for this?

Reply:
@joelhawksley the current `component_status_migrator` should handle this now. i think the layout component was a leftover from before i made those changes, and i didn't think to check it. 

🤔 probably worth writing a script to verify all the component links in `nav.yml` are correct, checking against the actual components' status. this could be a good CI check for builds